Formula/l/litmusctl.rb

```diff
@@ -0,0 +1,29 @@
+class Litmusctl < Formula
+  desc "Command-line interface for interacting with LitmusChaos"
+  homepage "https://litmuschaos.io"
+  url "https://github.com/litmuschaos/litmusctl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "363e58ecdf22dc9311bffde69e6f69a89bb30d77189791af9128f28e730b6ea6"
+  license "Apache-2.0"
+
+  head "https://github.com/litmuschaos/litmusctl.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Debug message
+    ohai "Building litmusctl..."
+    # Build the binary
+    ldflags = %W[
+      -s -w
+      -X main.CLIVersion=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "-o", bin/"litmusctl"
+    # Debug message
+    ohai "Installation complete."
+  end
+
+  test do
+    # Test installation
+    assert_match version.to_s, shell_output(bin/"litmusctl version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.